drupalcat/config/install/core.base_field_override.node.drupal_cat_project.promote.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 1c5bd5f0-fcbc-4721-ad01-1aed89e32022
```

Comment:
Existen más referencias de esto? El artículo es de 2013 y no lo deja muy claro, a parte de que parece que habla de compartir configuración entre sitios distintos. Cuando dice:

-field_uuid: 84e904df-2f14-46e8-9700-e00c5ca3f7d3
+field_name: field_blog_image
Note that this relies on the specific handling of the FieldInstance class and may not be a general solution.

parece indicar que cuando se borra el uuid se tiene que sustituir con otro identificador (en el ejemplo, el machine name del campo).

Un poco más preocupante es la afirmación:

Edit: also note (see comments) that it is important to include a hardcoded UUID if you want to avoid that your configuration items are seen by Drupal as different items between your development and production sites, causing configuration import/export between the two to fail.

Estaría bien ver una opinión más reciente de alguien más involucrado en CMI, o encontrar referencias fiables que afirmen que es una buena práctica borrar los uuids una vez exportada la configuración.

Si esa fuera la práctica estándar, debería ser fácil encontrar otras referencias. En un ejemplo de un site Drupal 8 que he econtrado los uuids están en el código: https://github.com/geerlingguy/demo-drupal-8/tree/master/sites/default/config/staging